NSSNamespace/User_Authentication
# touched new razor files for products and debugged.

To Test:

git fetch
git checkout fw-portProductsViews

Make sure that the following files have been added, and that there are no red squiggles:
1. Create.cshtml
2. Detail.cshtml
3. Index.cshtml
4. Type.cshtml
5. Types.cshtml
